app/api/resources/mentorship_relation.py

```diff
@@ -531,7 +531,7 @@ def post(cls, request_id):
         is_valid = CreateTask.is_valid_data(request_body)
 
         if is_valid != {}:
-            return is_valid, HTTPStatus.BAD_REQUEST
```

Comment:
I think this status was correct. because here this is a bad request because the request body is missing a field